internal/sqlite/migrations/202409061249_bootstrapDb/up.go

```diff
@@ -21,6 +21,7 @@ func (m *SqliteMigration) Up(grm *gorm.DB) error {
     	)`,
 		`CREATE TABLE IF NOT EXISTS transactions (
 			block_number INTEGER NOT NULL REFERENCES blocks(number) ON DELETE CASCADE,
+			block_time DATETIME NOT NULL,
```

Comment:
Since migrations are only run once, this needs to go in a brand new migration as an `alter table` statement